### ISSUE TITLE
Bug in PhononDOSAnalyser

### ISSUE THREAD

**Issue by marquezj** [mctools/ncrystal#187]
`PhononDOSAnalyser` fails when `fmt='anyvdos'` or `fmt=None`.

This is an example that shows the bug:

```
import NCrystal as NC
import numpy as np

hw = np.linspace(0,0.01, 100)
rho = hw**2

from NCrystal.misc import AnyVDOS
vdos = AnyVDOS((hw, rho), fmt='arrays')
dosana_raw = NC.PhononDOSAnalyser(vdos, fmt='anyvdos')
```

**Comment by tkittel** [comment]
Thanks, I will have a look!

**Comment by tkittel** [comment]
This is now fixed in NCrystal v3.9.4 (conda and PyPI package are being rebuilt as we speak, should be done in 20mins or so).

Be sure to update the minimum version check in any notebooks related to DOS work to 3.9.4.
